modules/ROOT/pages/_partials/commons/common-indexing.adoc

```diff
@@ -109,6 +109,183 @@ include::{root-partials}block_tabbed_code_example.adoc[]
 
 endif::is-c[]
 
+== Array Indexing 
 
+Couchbase Lite 3.2.1 introduces functionality to optimize querying arrays.
+Array `UNNEST` to unpack arrays within a document to allow joins with the parent object, and array indexes for indexing nested array properties and allow more efficient queries with `UNNEST`. 
```

Comment:
array indexes for indexing unnested array's values to allow more efficient queries with `UNNEST` ?

Reply:
Done